kernel/common/binding.rb

```diff
@@ -54,4 +54,12 @@ def eval(expr, filename=nil, lineno=nil)
 
     Kernel.eval(expr, self, filename, lineno)
   end
+
+  def local_variables
+    eval('local_variables')
```

Comment:
Yes, from the top of my head I think this can be reduced down to the following:

```
def local_variables
  scope = Rubinius::VariableScope.of_sender
  
  scope.local_variables
end
```

This can then be used in both `Kernel#local_variables` and `Binding#local_variables`.